packages/apps/reputation-oracle/server/src/common/constants/index.ts

```diff
@@ -1,7 +1,14 @@
+import { JobRequestType } from "../enums";
+
 export const NS = 'hmt';
 export const RETRIES_COUNT_THRESHOLD = 3;
 export const INITIAL_REPUTATION = 0;
 
+export const CVAT_RESULTS_ANNOTATIONS_FILENAME = 'resulting_annotations.zip'
+export const CVAT_VALIDATION_META_FILENAME = 'validation_meta.json'
```

Comment:
Are these file names always supposed to be `resulting_annotations.zip`, and `validation_meta.json`?

Reply:
For now yes, a bit further it will be changed @leric7